include/mapnik/expression_node.hpp

```diff
@@ -59,7 +59,10 @@ template <> struct make_op<mapnik::tags::logical_or> { using type =  std::logica
 template <typename Tag>
 struct unary_node
 {
-    unary_node (expr_node const& a)
+    unary_node (expr_node && a)
+        : expr(std::move(a)) {}
+
+    unary_node (expr_node const a)
```

Comment:
Probably `const&` was meant. Otherwise, this by-value conversion ctor would use `std::move` and effectively provide both `&` and `&&` ;)